etc/systemd/system/zfs-import-scan.service.in

```diff
@@ -4,13 +4,15 @@ Documentation=man:zpool(8)
 DefaultDependencies=no
 Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
+After=dracut-cmdline.service
 After=cryptsetup.target
 Before=zfs-import.target
 ConditionPathExists=!@sysconfdir@/zfs/zpool.cache
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
+ExecStartPre=@sbindir@/modprobe zfs zfs_autoimport_disable=1
```

Comment:
We actually removed this line about a year ago when a6cc97566c54255c98612c801f2c9d9be2d0e8f8 was merged.  Would it be possible to rely on the kernel module auto-loading in the dracut environment?

Reply:
`@sbindir@/modprobe` is wrong, anyways, as `@sbindir@` is zfs's sbindir which could be `/usr/local/sbin` for example. Use `/sbin` or better no path at all